tests/platforms/android/apk/test_verify_tools.py

```diff
@@ -0,0 +1,183 @@
+import subprocess
+from io import BytesIO
+from stat import S_IMODE
+from unittest import mock
+from zipfile import ZipFile
+
+import pytest
+from requests import exceptions as requests_exceptions
+
+from briefcase.exceptions import BriefcaseCommandError, NetworkFailure
+from briefcase.platforms.android.apk import ApkBuildCommand
+
+
+def create_sentinel_zipfile():
+    out = BytesIO()
+    with ZipFile(out, "w") as zipfile:
+        zipfile.writestr("tools/bin/sdkmanager", "")
+    return out.getvalue()
+
+
+@pytest.fixture
+def build_command(tmp_path, first_app_config):
+    class TestableApkBuildCommand(ApkBuildCommand):
+        dot_briefcase = tmp_path / ".briefcase" / "tools"
+
+    command = TestableApkBuildCommand(
+        base_path=tmp_path / "base_path", apps={"first": first_app_config},
+    )
+    command.tmp_path = tmp_path
```

Comment:
This *shouldn't* be needed; tmp_path will have a consistent value wherever it is used, so if a test case needs it, it can be added as a fixture argument for the test.

Reply:
Right -- I think that's vestigial. Let me remove that!

Reply:
Resolved in follow-up commit.